integration-test-apps/auto-instrumentation/flask/application.py

```diff
@@ -18,7 +18,7 @@
 # Setup AWS X-Ray Propagator
 
 # Propagators can be set using environment variable: OTEL_PROPAGATORS = aws_xray
```

Comment:
Just curious why not use the environment variable?

Reply:
No particular reason! It's just that the 2 main components you need to get OTel Python to work with X-Ray is that:
1. The IdsGenerator
2. The Propagator

So I wanted those to be obvious from the source code here in the manual instrumentation instead of hiding them in environment variables in the script.

You can do the ID Generator from [an environment variables](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-distro/src/opentelemetry/distro/__init__.py#L45) too as of recent I think!